src/main/osd/osd_elements.c

```diff
@@ -1062,6 +1062,8 @@ static void osdElementFlymode(osdElementParms_t *element)
         strcpy(element->buff, "ANGL");
     } else if (FLIGHT_MODE(ALT_HOLD_MODE)) {
         strcpy(element->buff, "ALTH");
+    } else if (FLIGHT_MODE(POS_HOLD_MODE)) {
+        strcpy(element->buff, "POSH");
```

Comment:
I have never seen POSH or ALTH on OSD, instead I see ANGL

Reply:
ANGLe mode is always enabled for ALT_HOLD_MODE / POS_HOLD_MODE. Either mode priority should be reversed(POS,ALT,ANG), or different modes mixed (AH+PH)

Reply:
Thanks, I've changed the sequence.